OpenRA.Mods.Cnc/OpenRA.Mods.Cnc.csproj

```diff
@@ -67,6 +67,7 @@
     <Compile Include="CncLoadScreen.cs" />
     <Compile Include="Traits\Attack\AttackPopupTurreted.cs" />
     <Compile Include="Traits\Buildings\ProductionAirdrop.cs" />
+    <Compile Include="Traits\Conditions\GrantConditionOnInfiltration.cs" />
```

Comment:
This should go to Infiltration folder imo.

Reply:
All the GrantCondition* traits so far have went into the Conditions folder, so notsure. If others also vote for that, I'll shift it over.

Reply:
Maybe rename to InfiltrateForCondition?

Reply:
IMO the current setup (`Conditions\GrantConditionOnInfiltration.cs`) is better.